app/SearchApi/SearchApi.Web/People/PersonSearchRequest.cs

```diff
@@ -46,5 +46,6 @@ public class PersonSearchRequest : Person
     public class DataProvider : ProviderProfile
     {
         public string Name { get; set; }
+        public DataProviderEnums DataProviderID { get; set; }
```

Comment:
Why do we have both? Shouldn't it be just the Provider Name (the string value of the enum?)